pkg/ebpf/events_pipeline.go

```diff
@@ -256,6 +255,14 @@ func (t *Tracee) decodeEvents(ctx context.Context, sourceChan chan []byte) (<-ch
 			evt.ProcessEntityId = utils.HashTaskID(eCtx.HostPid, eCtx.LeaderStartTime)
 			evt.ParentEntityId = utils.HashTaskID(eCtx.HostPpid, eCtx.ParentStartTime)
 
+			policies, err := policy.Snapshots().Get(eCtx.PoliciesVersion)
+			if err != nil {
+				t.handleError(err)
+				t.eventsPool.Put(evt)
+				continue
+			}
+			evt.Policies = unsafe.Pointer(policies)
```

Comment:
Why are you using the `unsafe.Pointer` instead of just a pointer to the policies?

Reply:
`types` would require importing the policy package to know about `*Policies`, which isn't the intention.

Reply:
But it is what you sort of do, no? Making the event to depend on the policies implementation.